r/tests/testthat/test-data-type.R

```diff
@@ -395,6 +395,18 @@ test_that("decimal type and validation", {
   expect_error(decimal(4, NA), '"scale" must be an integer')
 
   expect_r6_class(decimal(4, 2), "Decimal128Type")
+
+  expect_error(decimal128())
```

Comment:
Is this an error that the R package raises? If so, should we assert what it says?

Reply:
It's the generic R error: `Error in decimal128() : argument "precision" is missing, with no default`. I'd say not crucial to assert the actual content.